src/Path.Label.js

```diff
@@ -1,4 +1,20 @@
 L.Path.include({
+	showLabel: function () {
```

Comment:
Polygon and Polyline now have a `getCenter` method (which does not cover all cases, but works in the common situations).
Plus, I suggest to add a `latlng` parameter to `showLabel`, with a fallback to `getCenter`, so it's easier for users to give the position they want when using a static label on a path, and it still works for basic cases.